query-grammar/src/query_grammar.rs

```diff
@@ -1537,7 +1539,9 @@ mod test {
     fn test_parse_query_with_range() {
         test_parse_query_to_ast_helper("[1 TO 5]", "[\"1\" TO \"5\"]");
         test_parse_query_to_ast_helper("foo:{a TO z}", "\"foo\":{\"a\" TO \"z\"}");
+        test_parse_query_to_ast_helper("foo: \t{a TO z}", "\"foo\":{\"a\" TO \"z\"}");
```

Comment:
not sure we want to support tabs in the query parser

Reply:
I just added it to highlight this, but since we are using `space0` everywhere already, I think we already support tabs specifically where we support whitespace generally.